src/main/java/it/gov/pagopa/pu/processecexutions/enums/IngestionFlowFileVersion.java

```diff
@@ -0,0 +1,37 @@
+package it.gov.pagopa.pu.processecexutions.enums;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import java.util.Arrays;
+
+@Schema(enumAsRef = true)
+public enum IngestionFlowFileVersion {
```

Comment:
These cannot be generic!
```suggestion
public enum DpInstallmentsIngestionFlowFileVersion {
```